jep/7/README.adoc

```diff
@@ -132,11 +183,106 @@ Workarounds for this include downloading affected plugins from Artifactory, and
 
 If previous core compatibility fixes are reverted, or future core changes break ruby-runtime, users of those plugins will be impacted.
 
+=== Affected plugins, Ruby runtime
+
+Below you can find a list of the affected plugins which are/were being hosted in the main Jenkins update center.
+There might be other 3rd-party plugins affected.
+
+Gitlab Hook::
+Last released **5 years ago**. +
+Contains multiple security vulnerabilties. +
+Suggestion: Use the plugin:gitlab-plugin[GitLab] plugin and the plugin:gitlab-branch-source[GitLab Branch Source] plugin.
```

Comment:
All of these suggestions have no reason to be here.

Reply:
Why not? It may help someone reading the doc

Reply:
It's a specification, not a user guide. Different documents for different audiences.